docs/logs/getting-started/Program.cs

```diff
@@ -26,7 +26,26 @@ public static void Main()
         });
         var logger = loggerFactory.CreateLogger<Program>();
 
+        // unstructured log
```

Comment:
should we say - not recommended?

Reply:
It has to come from the official ILogger document instead of here.

Reply:
Put a link ti ILogger doc, so the user could get which is recommended or not if start from this sample program?

Reply:
This is no good document on ILogger today and it is a TODO for .NET team.